### ISSUE TITLE
Dataset and Dataloader for custom point cloud dataset

### ISSUE THREAD

**Issue by davodogster** [torch-points3d/torch-points3d#716]
Hello,
Awesome work on the repo!

I have individual point cloud files (e.g a "per object" .txt or .las file containing the X,Y,Z's and a separate file for the target - It's a multitask problem where each point cloud has 2 associated labels.
Could you give me any pointers on how I can  convert  this data into a PyTorch dataset or use it in a Dataloader for 3D Deep Learning?
To make things more complicated the point clouds all contain a different number of points.

I have been searching issues and looking for information but haven't come across much for this. Every repo for images and point clouds I would have thought the most important thing would be "train on custom dataset" or "dataloader for custom dataset".

Many Thanks! @nicolas-chaulet

**Comment by CCInc** [comment]
Our datasets are based on pytorch geometric datasets.  You can get an overview here: https://pytorch-geometric.readthedocs.io/en/latest/notes/create_dataset.html
